packages/perseus/src/widgets/radio/__stories__/base-radio.stories.jsx

```diff
@@ -13,28 +13,63 @@ export default ({
     title: "Perseus/Widgets/Radio/Base Radio",
 }: Story);
 
-const defaultObject = {
+function generateChoice(options) {
```

Comment:
I copied this function and I'm embarrassed, but the tests are so far from the stories I didn't know the logical place to have a shared helper. I think this could be an argument for refactoring our file structure to have components closer to their tests/stories.